rotkehlchen/tests/api/test_addressbook.py

```diff
@@ -542,43 +542,59 @@ def names_request(chain_addresses: list[OptionalChainAddress]) -> requests.Respo
                     {'address': chain_address.address, 'blockchain': chain_address.blockchain.serialize() if chain_address.blockchain is not None else None} for chain_address in chain_addresses],  # noqa: E501
             },
         )
-    address_rotki = to_checksum_address('0x9531c059098e3d194ff87febb587ab07b30b1306')
-    address_cody = to_checksum_address('0x368B9ad9B6AAaeFCE33b8c21781cfF375e09be67')
-    address_rose = to_checksum_address('0xE07Af3FBEAf8584dc885f5bAA7c72419BDDf002D')
-    address_tylor = to_checksum_address('0xC88eA7a5df3A7BA59C72393C5b2dc2CE260ff04D')
-    address_nonlabel = to_checksum_address('0x9d904063e7e120302a13c6820561940538a2ad57')
-    address_1world = to_checksum_address('0xfDBc1aDc26F0F8f8606a5d63b7D3a3CD21c22B23')
-    address_firstblood = to_checksum_address('0xAf30D2a7E90d7DC361c8C4585e9BB7D2F6f15bc7')
-    address_kraken10 = to_checksum_address('0xAe2D4617c862309A3d75A0fFB358c7a5009c673F')
 
+    address_rotki = string_to_evm_address('0x9531C059098e3d194fF87FebB587aB07B30B1306')
```

Comment:
rewrote them all as **constant** checksummed addresses. Spent too much time debugging that those constants were not checksummed before.